core/src/main/scala/com/dimafeng/testcontainers/TestLifecycleAware.scala

```diff
@@ -1,4 +1,4 @@
-package com.dimafeng.testcontainers.lifecycle
+package com.dimafeng.testcontainers
```

Comment:
Why is this package changed?

Reply:
It seems like it happened accidentally. Good catch 👍